### ISSUE TITLE
Ck Crashing during FaceGen

### ISSUE THREAD

**Issue by Dougal123** [Perchik71/Creation-Kit-Platform-Extended#74]
Hi, So I was using the progam https://www.nexusmods.com/fallout4/mods/86374 (Vault tec face gen) that uses xedit + CK to generate faces for FO4. The Xedit script completes and then it opens CK and shortly after it crashes with the error stating 

xception:
    Code: C0000005
    Address: 00007FF76D4FA336
    Message: The thread tried to read from or write to a virtual address for which it does not have the appropriate access.

[Creationkit.exe_20241112_182129.zip](https://github.com/user-attachments/files/17721281/Creationkit.exe_20241112_182129.zip)

Thank you for your time.

**Comment by Perchik71** [comment]
This is stated on the mod page itself (Vault tec face gen), enable bBSPointerHandleExtremly.

**Comment by evildarkarchon** [comment]
Here's some more reading material for you. I'm having a similar, if not identical, issue.

[Creationkit.exe_20250304_093747.zip](https://github.com/user-attachments/files/19067868/Creationkit.exe_20250304_093747.zip)

[CreationKitPlatformExtended.ini.zip](https://github.com/user-attachments/files/19067879/CreationKitPlatformExtended.ini.zip)

**Comment by Perchik71** [comment]
@evildarkarchon No, the error is different, you have an error in the string storage (Cache). 
It is very likely that you have exceeded the limit. Since a entry string is expected, there it is **0032004100360030** so doesn't look like character codes.

![Image](https://github.com/user-attachments/assets/0fd9840c-7a5e-4935-ae78-726b2fe298cb)
![Image](https://github.com/user-attachments/assets/4ac02ceb-95a0-4f41-a840-3dc940214d09)
